scripts/deploy.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -e -u -x
+set -o pipefail
+source /opt/rpc-extras/os-ansible-deployment/scripts/scripts-library.sh
+
+export RPCD_LOGGING=${RPCD_LOGGING:-"FALSE"}
+export RPCD_AIO=${RPCD_AIO:-"FALSE"}
+
+OSAD_DIR='/opt/rpc-extras/os-ansible-deployment'
+RPCD_DIR='/opt/rpc-extras/rpcd'
+
+# setup the things
```

Comment:
For the idempotent type bits that have been added for bootstrap-aio.sh and bootstrap-ansible.sh - would that not be better done upstream in those scripts instread of here?